OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -3111,17 +3109,13 @@ public static void sendOutcomeWithValue(@NonNull String name, float value) {
    }
 
    public static void sendOutcomeWithValue(@NonNull String name, float value, OutcomeCallback callback) {
-      if (!validateOutcomeEntry(name))
+      if (!isValidOutcomeEntry(name))
          return;
 
       outcomeEventsController.sendOutcomeEventWithValue(name, value, callback);
    }
 
-   private static boolean validateOutcomeEntry(String name) {
-      if (outcomeEventsController == null) {
```

Comment:
We still want to have the `outcomeEventsController` `null` check here. As init or any of our services / broadcastReceiver  may not have been called to setup a context.

In the future we can have it go through `addTaskToQueue` like we have for `sendTags` so the call isn't drop if the dev calls things out of order.